src/Library/demos/Column View/main.js

```diff
@@ -0,0 +1,101 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+
+const column_view = workbench.builder.get_object("column_view");
+
+function createFirstCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
+      margin_start: 12,
+      margin_end: 12,
+    });
+    listItem.set_child(label);
+  });
+  factory.connect("bind", (factory, listItem) => {
+    const labelWidget = listItem.get_child();
+    const modelItem = listItem.get_item();
+    labelWidget.label = modelItem.get_name();
+  });
+  return columnViewColumn;
+}
+
+function createSecCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
+      margin_start: 12,
+      margin_end: 12,
+    });
+    listItem.set_child(label);
+  });
+  factory.connect("bind", (factory, listItem) => {
+    const labelWidget = listItem.get_child();
+    const modelItem = listItem.get_item();
+    labelWidget.label = modelItem.get_size().toString();
+  });
+  return columnViewColumn;
+}
+
+function createThirdCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
+      margin_start: 12,
+      margin_end: 12,
+    });
+    listItem.set_child(label);
+  });
+  factory.connect("bind", (factory, listItem) => {
+    const labelWidget = listItem.get_child();
+    const modelItem = listItem.get_item();
+  });
```

Comment:
In this column, we can use `Gio.FileInfo.get_modification_date_time()`, just be sure to set the attributes to `"standard::*,time:modified"` so that attribute is available.

Then something simple like this can go in the label:

```js
const date = modelItem.get_modification_date_time();
labelWidget.label = date.format("%F");
```

Reply:
I get date as null ; I also tried using ```fileInfo.get_attribute_uint64()``` but still it shows empty.